website/views/shared/footer.jade

```diff
@@ -95,6 +95,8 @@ footer.footer(ng-controller='FooterCtrl')
             a.btn.btn-default(ng-click='setHealthLow()') Health = 1
             a.btn.btn-default(ng-click='addMissedDay(1)') +1 Missed Day
             a.btn.btn-default(ng-click='addMissedDay(2)') +2 Missed Days
+            a.btn.btn-default(ng-click='addMissedDay(8)') +8 Missed Days
```

Comment:
Perhaps it would be better to add a button to set the last cron to a specific day, rather than adding in these extra buttons.